### PR TITLE
chore(modules): exposing the net class

### PR DESCRIPTION
**Pull request by gergelyke**
Hey,

PR for #5758 - one question though. I couldn't find any unit/integration tests, did I miss something?

### CODE REVIEW

File: drivers/javascript/net.coffee [hunk 1]
```diff
@@ -1665,3 +1665,8 @@ module.exports.connect = varar 0, 2, (hostOrCallback, callback) ->
                 resolve(result)
         create_connection(host, wrappedCb)
     ).nodeify callback
+
+# Exposing the mail classes
```
Comment (tothandras) [inline drivers/javascript/net.coffee]: typo?

**Comment by danielmewes** [comment]
Thanks for opening the PR @gergelyke .

You can run the JavaScript tests by `test/rql_test/test-runner -i js`. You'll need to compile the RethinkDB server before you do this, or alternatively pass in the location of the `rethinkdb` binary to use through the `-b` parameter.

@deontologician Could you take a look at @gergelyke 's patch when you have a few minutes?

**Comment by deontologician** [comment]
This seems reasonable. @larkost can you remember the thing we exported with an underscore previously? We'd talked about exporting something ( I think it was the Connection class) to make testing easier, and landed on having it "privately" exported. I may be remembering something else though

**Comment by larkost** [comment]
@deontologician I think you might be remembering either that we expose `_r` on the `Connection` class in Python, or that we export `_bluebird` on the `rethinkdb` object (the only thing we currently export) in the JavaScript driver.

**Comment by deontologician** [comment]
That was it, thanks.

I am :+1: for merging this

**Comment by danielmewes** [comment]
Back-ported into v2.3.x with 4b5d47a. This will ship with the next driver release.

**Pull request merged**